docs/pages/attributesJSON.rst

```diff
@@ -0,0 +1,194 @@
+
+"physics simulator"
+	- string
+	- What physics engine should be used for dynamics simulation.  Currently supports "bullet" for Bullet physics simulation, and "none", meaning kinematic motion is to be used.
+"gravity"
+	- 3-vector
+	- Gravity to use for physical modeling. This can be overridden by Stage attributes.
+"timestep"
+	- double
+	- The timestep to use for forward simulation.
+"friction coefficient"
+	- double
+	- The coefficient of friction. This can be overridden in Stage and Object Attributes.
+"restitution coefficient"
+	- double
+	- The coefficient of restitution. This can be overridden in Stage and Object Attributes.
+"rigid object paths"
+	- list of strings
+	- A list of locations to query for supported object files that should be available to be loaded into the world.
+
+`Stage Attributes`_
+===================
+A stage in Habitat-Sim is a static object consisting of background scenery wherein an agent acts.  Stage Attributes templates hold relevant information describing a stage's mesh, geometry and physical properties.  Any source configuration files used to build these attributes should be named in the format :
+
+ 	<stagename>.stage_config.json
+
+An example of an appropriate configured file can be found below :
+
+.. include:: ../../data/test_assets/scenes/stage_floor1.stage_config.json
+    :code: json
+
+
+Stage Mesh Handles And Types
+----------------------------
+
+Below are the handles and descriptors for various mesh assets used by a stage.
+
+"render mesh"
+	- string
+	- The name of the file describing the render mesh to be used by the stage.
+"collision mesh"
+	- string
+	- The name of the file describing the collision mesh to be used by the stage.
+"semantic mesh"
+	- string
+	- The name of the file describing the stage's semantic mesh.
+"house filename"
+	- string
+	- The name of the file containing semantic type maps and hierarchy.
+"nav mesh"
+	- string
+	- The name of the file describing the NavMesh for this stage.
+
+Stage Frame and Origin
+----------------------
+
+The tags below are used to build a coordinate frame for the stage, and will override any default values set based on render mesh file name/extension.  If either **"up"** or **"front"** are specified, both must be provided and they must be orthogonal.
+
+"up"
+	- 3-vector
+	- Describes the **up** direction for the stage.
+"front"
+	- 3-vector
+	- Describes the **forward** direction for the stage.
+"origin"
+	- 3-vector
+	- Describes the **origin** of the stage, for alignment purposes.
+
+Stage Physics and Object-related Parameters
+-------------------------------------------
+
+Below are stage-specific physical and object-related quantities.  These values will override similarly-named values specified in the Physics Manager Attributes.
+
+"scale"
+	- 3-vector
+	- The default scale to be used for the stage.
+"gravity"
+	- 3-vector
+	- Gravity to use for physical modeling.
```

Comment:
Before or after coordinate system correction? (In what coordinate frame?)

Reply:
Gravity is in world frame, to the best of my knowledge.